std/math.d

```diff
@@ -7291,3 +7292,59 @@ unittest
         }
     }
 }
+
+/**
+Gives the next highest power of two after $(D val). $(T) can be any non-floating
+point numerical type. If the input would result in an overflow, then zero is
```

Comment:
"any non-floating point numerical type"? That's ambiguous, might imply e.g. `BigInt` or `char`... Why not just "built-in integral type"?